browser/brave_content_browser_client.cc

```diff
@@ -143,6 +148,21 @@ bool BraveContentBrowserClient::AllowAccessCookie(
       BraveShieldsWebContentsObserver::GetTabURLFromRenderFrameInfo(
           render_process_id, render_frame_id, -1).GetOrigin();
   ProfileIOData* io_data = ProfileIOData::FromResourceContext(context);
+  bool allow = false;
```

Comment:
is it correct that the we wanted to avoid a code duplication in this method? Is it the subject of some follow-up?